cdap-client/src/main/java/co/cask/cdap/client/StreamClient.java

```diff
@@ -85,7 +87,7 @@ public StreamClient(ClientConfig config) {
    * Gets the configuration of a stream.
    *
    * @param streamId ID of the stream
-   * @throws IOException if a network error occurred
+   * @throws IOException             if a network error occurred
```

Comment:
That's not the convention we have in the code base. Can you revert those changes in this file?